src/components/TextInput.tsx

```diff
@@ -26,17 +26,28 @@ const Asterix = styled(Label)`
 `;
 
 const Input = styled.input<{ error: boolean }>`
-	border-radius: 4px;
-	font-size: 0.8em;
 	border: none;
-	width: 100%;
+	border: 1px solid #bdbdbd;
+	border-radius: 10px;
+	outline: none;
+	// outline: auto 1px #bdbdbd;
+	font-size: 0.8em;
+	color: black;
+	width: 10em;
+	height: 2em;
 	padding: 0.25em;
+	&:focus {
+		border: 1px solid #aaaaaa;
+	}
+	&::placeholder {
+		color: #bdbdbd;
+	}
 	${(props) =>
 		props.error &&
 		css`
-			outline: auto 4px red; // Need to pick an error colour?
+			border: 2px solid #ff8888; // Need to pick an error colour?
 			&:focus {
-				outline: auto 4px red;
+				border: 2px solid red;
```

Comment:
Do we want different focus and regular colors?

Reply:
Didn't even remember changing the original color - what do you think?

Reply:
Choose whichever color you think looks best, and if you think there is a reason to have the focus border different than the standard border you can leave that in, otherwise make them the same.